content/dns.md

```diff
@@ -204,8 +204,21 @@ BOSH DNS provides its own structured query language for querying instances
 based on an instance's endemic and organizational relationship; e.g., by an
 instance's healthiness, its availability zone, or group id.
 
-An example of a DNS query is as follows:
+A few example DNS queries:
 
+* `dig @bosh-dns q-s3.zookeeper.*.zk-prod.bosh`
+
+Query for BOSH instances that are healthy (`q-s3`), from instance group `zookeeper`, on all networks (`*`), in deployment `zk-prod`.
+
+* `dig @bosh-dns q-s1-a2.diego-cell.*.*.bosh`
```

Comment:
Did you mean for this to be q-s1.q-a2...

```suggestion
* `dig @bosh-dns q-s1.q-a2.diego-cell.*.*.bosh`
```

Reply:
Earlier today I _thought_ I tried the query you're suggesting (which is why my explanation of the query has `q-a2` in it). Now, I can't get it to return instances:

```
$ dig @169.254.0.2 +short q-s4.q-a2.diego-cell.*.*.bosh
 <no response>

$ dig @169.254.0.2 +short q-s4-a2.diego-cell.*.*.bosh
10.0.1.29
10.0.1.28
```
This is on a cf-deployment environment with BOSH `Version   268.0.1` and bosh-dns-release 1.10.0.

Should I just update my explanation of the query to be `a2` instead of `q-a2`?

Reply:
Ah ok; thanks for explaining. I believe that's what you have already, but yeah feel free to update what you have tested and is working.